### PR TITLE
Updated the Apache Hadoop dashboard and alert.

### PR DESCRIPTION
**Pull request by pkudikyala**
**Ticket:** https://issues.newrelic.com/browse/NR-145218
**Description:** Updated the dashboard about the section and logo.

# Summary

A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. **If it is not ready, please mark the pull request as a draft.**

_The owners of this repo are not experts in the subject matter of the quickstarts. We review for the quickstart to be functional and for security risks. If you are seeking feedback on the content of the quickstart, please seek out a subject matter expert. If you are not an internal NR contributor, we can do our best to connect you with a content reviewer._

## Pre merge checklist

- [ ] Did you check you NRQL syntax? - Does it work?
- [ ] Did you include a Data source and Documentation reference?
- [ ] Are all documentation links publicly accessible?
- [ ] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [ ] Did you check your descriptive content for spelling and grammar errors?
- [ ] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [ ] Does the PR contain a screenshot for each of your dashboards?
- [ ] Do your screenshots show data?
- [ ] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [ ] Did you check that your alerts actually work?

### CODE REVIEW

**Review by pkosanam — DISMISSED** [review]
LGTM

File: alert-policies/apache-hadoop/UsedDiskPercentage.yml
Comment (mdumpati) [inline alert-policies/apache-hadoop/UsedDiskPercentage.yml:1]: @pkudikyala In this Used Disk Percent alert section can you change the description to " This alert is triggered when disk usage exceeds 90% for at least 5 minutes." (in line number 6)

**Review by mdumpati — COMMENTED** [review]
@pkudikyala Check my comments

File: alert-policies/apache-hadoop/UsedDiskPercentage.yml
Comment (pkudikyala) [reply alert-policies/apache-hadoop/UsedDiskPercentage.yml:1]: Hi @mdumpati, Updated the alert description as per your suggestion. Kindly review it.
 
Thanks.

**Review by mdumpati — APPROVED** [review]
LGTM

**Review by seemantk — APPROVED** [review]

**Pull request merged**